opba-protocols/xs2a-protocol/src/main/resources/processes/xs2a/consent/xs2a-authorize-consent.bpmn20.xml

```diff
@@ -68,7 +61,12 @@
     <serviceTask id="callFintechWebhook" name="Calls FinTech webhook to notify completed authorization" flowable:delegateExpression="${xs2aReportToFintechConsentAuthorized}"></serviceTask>
     <sequenceFlow id="sid-7985590B-669B-4877-AFEF-B3BD5824C52B" sourceRef="callFintechWebhook" targetRef="end"></sequenceFlow>
     <sequenceFlow id="sid-59AE4CB4-9545-4CD9-9495-6A5D8AF74CBE" sourceRef="persistConsentAndContext" targetRef="callFintechWebhook"></sequenceFlow>
-    <sequenceFlow id="sid-DCA3C0C2-1C4E-404E-8C08-582FDD3F6643" name="Implicitly handled by xs2aDenyAuthorization" sourceRef="sid-7E036180-B980-4A2D-8B6C-7407E2C90C2A" targetRef="end"></sequenceFlow>
+    <sequenceFlow id="isOkConsent" name="Consent OK" sourceRef="sid-7E036180-B980-4A2D-8B6C-7407E2C90C2A" targetRef="persistConsentAndContext">
+      <conditionExpression xsi:type="tFormalExpression"><![CDATA[#{xs2aConsentInfo.isOkRedirectConsent(CONTEXT)}]]></conditionExpression>
+    </sequenceFlow>
+    <sequenceFlow id="sid-DCA3C0C2-1C4E-404E-8C08-582FDD3F6643" name="Implicitly handled by xs2aDenyAuthorization" sourceRef="sid-7E036180-B980-4A2D-8B6C-7407E2C90C2A" targetRef="sid-xs2aReportToFintechConsentRevoked"></sequenceFlow>
+    <serviceTask id="sid-xs2aReportToFintechConsentRevoked" name="Calls FinTech webhook to notify  revoke authorization" flowable:delegateExpression="${xs2aReportToFintechConsentRevoked}"></serviceTask>
```

Comment:
This service task should be asynchronous `flowable:async="true"` because synchronous service tasks rollback entire process state until the last asynchronous element (work in one large transaction with all elements - so that intermediate state of rollback can't be seen)

Reply:
done